src/py42/sdk/queries/fileevents/filters/risk_filter.py

```diff
@@ -0,0 +1,232 @@
+        - :attr:`RiskIndicator.EmailServiceUploads.ICLOUD`
+        - :attr:`RiskIndicator.EmailServiceUploads.MAIL_DOT_COM`
+        - :attr:`RiskIndicator.EmailServiceUploads.OUTLOOK`
+        - :attr:`RiskIndicator.EmailServiceUploads.PROTONMAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.QQMAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.SINA_MAIL`
+        - :attr:`RiskIndicator.EmailServiceUploads.SOHU_MAIl`
+        - :attr:`RiskIndicator.EmailServiceUploads.YAHOO`
+        - :attr:`RiskIndicator.EmailServiceUploads.ZOHO_MAIL`
+        - :attr:`RiskIndicator.RemovableMedia.AIRDROP`
+        - :attr:`RiskIndicator.RemovableMedia.REMOVABLE_MEDIA`
+        - :attr:`RiskIndicator.FileCategories.AUDIO`
+        - :attr:`RiskIndicator.FileCategories.DOCUMENT`
+        - :attr:`RiskIndicator.FileCategories.EXECUTABLE`
+        - :attr:`RiskIndicator.FileCategories.IMAGE`
+        - :attr:`RiskIndicator.FileCategories.PDF`
+        - :attr:`RiskIndicator.FileCategories.PRESENTATION`
+        - :attr:`RiskIndicator.FileCategories.SCRIPT`
+        - :attr:`RiskIndicator.FileCategories.SOURCE_CODE`
+        - :attr:`RiskIndicator.FileCategories.SPREADSHEET`
+        - :attr:`RiskIndicator.FileCategories.VIDEO`
+        - :attr:`RiskIndicator.FileCategories.VIRTUAL_DISK_IMAGE`
+        - :attr:`RiskIndicator.FileCategories.ZIP`
+        - :attr:`RiskIndicator.MessagingServiceUploads.FACEBOOK_MESSENGER`
+        - :attr:`RiskIndicator.MessagingServiceUploads.MICROSOFT_TEAMS`
+        - :attr:`RiskIndicator.MessagingServiceUploads.SLACK`
+        - :attr:`RiskIndicator.MessagingServiceUploads.WHATSAPP`
+        - :attr:`RiskIndicator.Other.OTHER`
+        - :attr:`RiskIndicator.MessagingServiceUploads.UNKNOWN`
+        - :attr:`RiskIndicator.SocialMediaUploads.FACEBOOK`
+        - :attr:`RiskIndicator.SocialMediaUploads.LINKEDIN`
+        - :attr:`RiskIndicator.SocialMediaUploads.REDDIT`
+        - :attr:`RiskIndicator.SocialMediaUploads.TWITTER`
+        - :attr:`RiskIndicator.UserBehavior.FILE_MISMATCH`
+        - :attr:`RiskIndicator.UserBehavior.OFF_HOURS`
+        - :attr:`RiskIndicator.UserBehavior.REMOTE`
+    """
+
+    _term = u"riskIndicatorNames"
+
+    @staticmethod
+    def choices():
+        return (
+            RiskIndicator.CloudDataExposures.choices()
+            + RiskIndicator.CloudStorageUploads.choices()
+            + RiskIndicator.CodeRepositoryUploads.choices()
+            + RiskIndicator.EmaiServiceUploads.choices()
+            + RiskIndicator.ExternalDevices.choices()
+            + RiskIndicator.FileCategories.choices()
+            + RiskIndicator.MessagingServiceUploads.choices()
+            + RiskIndicator.Other.choices()
+            + RiskIndicator.SocialMediaUploads.choices()
+            + RiskIndicator.UserBehavior.choices()
+        )
+
+    class CloudDataExposures(object):
+        PUBLIC_CORPORATE_BOX = "Public link from corporate Box"
```

Comment:
These are some strange enums, I am guessing this is what you meant when you said these were not going to change? I wish we could have had them be something else still.

Reply:
Yeah that's exactly what I was talking about, I double checked  and they're not tied to the UI and are not going to change, but I agree they're strange.